pkg/webui/console/components/webhook-form/index.js

```diff
@@ -74,6 +74,32 @@ export default class WebhookForm extends Component {
     this.form = React.createRef()
   }
 
+  static propTypes = {
+    appId: PropTypes.string,
+    initialWebhookValue: PropTypes.shape({
+      ids: PropTypes.shape({
+        webhook_id: PropTypes.string,
+      }),
+    }),
+    onDelete: PropTypes.func,
```

Comment:
```suggestion
    onDelete: PropTypes.func.isRequired,
```

Reply:
this change also results in PropType error